addons/mail/sfu_server/README.md

```diff
@@ -0,0 +1,158 @@
+# Odoo: Discuss SFU
+
+## Overview
+
+Contains the code for an SFU (Selective Forwarding Unit) server 
+to be used alongside the Odoo Discuss module. The SFU server is responsible for
+handling the WebRTC connections between users and providing channels to coordinate
+these connections.
+
+## Prerequisites
+
+- An Odoo server with the Discuss module installed
+- [Node.js](https://nodejs.org/)
+
+## Before deployment
+
+Build the client bundle
+
+ ```bash
+     npm install
+     npm run build
+ ```
+
+Once the bundle is built, it should be added to the libraries of the Odoo server that will use
+this version of the server.
+
+## Deployment
+
+1. Install dependencies.
+    ```bash
+        npm ci -omit=dev
+    ```
+2. Run the server.
+    ```bash
+        npm PROXY=SSL PUBLIC_IP=134.123.222.111 AUTH_KEY=u6bsUQEWrHdKIuYplirRnbBmLbrKV5PxKG7DtA71mng= run start
+    ```
+
+The available environment variables are:
+
+- **PUBLIC_IP** (required): used to establish webRTC connections to the server
+- **AUTH_KEY** (required): the base64 encoded encryption key used for authentication
+- **HOSTNAME**:  HTTP/WS hostname, defaults to "0.0.0.0" (listen on all interfaces)
+- **PORT**: port for HTTP/WS, defaults to standard ports
+- **RTC_IP**: IP address for RTC, defaults to the HTTP IP, defaults to "0.0.0.0"
```

Comment:
I don't get this parameter. Is it supposed to be the listening interface? In that case it's probably always going to be 0.0.0.0, because it won't go through a reverse proxy anyway.

Reply:
Maybe call it `RTC_INTERFACE`?